regression/contracts/assigns_enforce_arrays_02/test.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+KNOWNBUG
```

Comment:
A. If this was working before, has this PR broken it?
B. If it wasn't then why did the test pass?

Reply:
Fixed bug.